docs/data/data-grid/aggregation/aggregation.md

```diff
@@ -6,9 +6,9 @@ title: Data Grid - Aggregation
 
 <p class="description">Add aggregation functions to the Data Grid so users can combine row values.</p>
 
-You can aggregate rows through the data grid interface by opening the column menu and selecting from the items under **Aggregation**.
+You can aggregate rows through the Grid interface by opening the column menu and selecting from the items under **Aggregation**.
 
-The aggregated values are rendered in a footer row at the bottom of the data grid.
+The aggregated values are rendered in a footer row at the bottom of the Grid.
```

Comment:
Before @samuelsycamore PR we were using `grid`
Someone changed it for `data grid` in a lot of places whereas @samuelsycamore changes it for `Grid`.

@samuelsycamore @cherniavskii if we can align on one nomenclature for this very common scenario :+1:

Reply:
We have a [Grid](https://mui.com/material-ui/react-grid/) component in material-ui too, I'm thinking if we should distinguish between the two (MUI X Data Grid and MUI Grid) in our docs, considering the core and X docs are not separated by user's perspective. 🤔

Reply:
I did not update all the usages of `data grid` in this PR but we can do it in a follow up :+1: